pennylane/operation.py

```diff
@@ -1361,15 +1361,16 @@ def adjoint(self) -> "Operator":  # pylint:disable=no-self-use
         raise AdjointUndefinedError
 
     def expand(self):
-        """Returns a tape that has recorded the decomposition of the operator.
+        """Returns a tape that contains the decomposition of the operator.
 
         Returns:
             .QuantumTape: quantum tape
         """
         if not self.has_decomposition:
             raise DecompositionUndefinedError
 
-        qscript = qml.tape.make_qscript(self.decomposition)()
+        with QueuingManager.stop_recording():
+            qscript = qml.tape.QuantumScript(self.decomposition())
```

Comment:
And just to clarify, this is only to help us find where the problems are and won't be a change in this PR.

Reply:
Not in this PR.